incubator/virtualcluster/config/rbac/auth_proxy_service.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     control-plane: controller-manager
     controller-tools.k8s.io: "1.0"
   name: controller-manager-metrics-service
-  namespace: system
+  namespace: kube-system
```

Comment:
change to virtualcluster-system

Reply:
I think we should keep it this way. This service is for meta cluster.

Reply:
ok